c2corg_ui/static/partials/cards/images.html

```diff
@@ -0,0 +1,16 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <img ng-src="{{cardCtrl.createImg()}}" alt="{{locale['title']}}">
```

Comment:
Are the cards that big?! :p 

OK let's try with MI but honestly I'm afraid those files are a bit too heavy for a list page.